Real_world_examples/Radar_water_detection.ipynb

```diff
@@ -2309,11 +2309,663 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ds_s1["vh_dB"] = 10 * np.log10(ds_s1.vh)
</pre><p>I think <a href="https://github.com/geoscience-aman" target="_blank">@geoscience-aman</a> hada function for applying this step? If so, could be good to use that here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>